### ISSUE TITLE
List of OBJKTs you have currently listed for sale (from your collection)

### ISSUE THREAD

**Issue by xmannm** [hicetnunc2000/hicetnunc#700]
As a collector when I own only one edition of some artwork and I list it for sale, it's gone from my collection and I'm never able to find that piece ever again. 

I could imagine some kind of list of OBJKTs you have currently listed for sale. (in the menu or somewhere in the Collection tab)
You could easier check the listed objects and keep a track of them. 

It could either be a page with images of the objects (just like your Creations and Collection page) or just a list of links to them would be very helpful.

**Comment by kylegrover** [comment]
I agree this should be part of the main website. For now I use an external tool I made which lists your pending swaps: https://ufffd.com/hicettum/swaps

**Comment by xmannm** [comment]
Thanks for the quick response and even more for sending the tool, I
couldn't find anything like that!

Dne so 15. kvě 2021 23:58 uživatel Kyle Grover ***@***.***>
napsal:

**Comment by andrevenancio** [comment]
This feature is already implemented. I'm waiting on @crzypatchwork to push it to production.

**Comment by xmannm** [comment]
That is amazing news, thank you!

**Comment by violetforest** [comment]
duplicate of https://github.com/hicetnunc2000/hicetnunc/issues/276
